.eslintrc

```diff
@@ -8,6 +8,7 @@
     "moduleForComponent": "off",
     "testStart": "off",
     "testDone": "off",
-    "sinon": "off"
+    "sinon": "off",
+    "visit": "off",
```

Comment:
this should probably be in shared eslint config, but I will do it later as plugins will have to be updated too

Reply:
This was my mistake - I was used it to debug locally and find all occurrences. It causes test failures and I'm putting it back.